examples/impute/plot_multiple_imputation.py

```diff
@@ -0,0 +1,359 @@
+:class:`sklearn.linear_model.BayesianRidge` predictions. If each
+:class:`sklearn.impute.IterativeImputer` uses a different `random_state`, this
+results in multiple imputations, each of which can be used to train a
+predictive model.
+
+One final model is obtained by combining the estimates of each model with
+Rubin's pooling rules, which are as follows. The overall point estimate after
+multiple imputation (denoted by Qbar) is the average of all the m point
+estimates. The variance of the overall point estimate is a combination of
+so-called within imputation variance (Ubar) and between imputation
+variance (B). Ubar is the average of the m variances of the m point estimates.
+Both Qbar and Ubar are corrected with a factor 1 / m to account for sampling
+variance. The between imputation variance (B) is the sum of the squared
+difference between Qbar and the m point estimates, corrected with a factor
+1 / (m – 1). Then, the total variance (T) of the MI overall point estimate is
+Ubar + B + B/m.
+
+These rules assume that the parameters of interest are normally distributed
+which is the case with, for example, estimates of the mean and regression
+coefficients. Other parameters, such as correlation coefficients need
+transformation to suit the assumption of normality. If it is not possible to
+approximate a normal distribution, it is better to use robust summary measures
+such as medians or ranges instead of using Rubin's pooling rules. This applies
+to an estimate like explained variance.
+
+In this Example we show how to use :class:`sklearn.impute.IterativeImputer`
+to perform multiple imputation. In Example 1 we show the effect of Rubin’s
+pooling rules on the variance of regression estimates. Due to the between
+imputation variance, the standard errors of all regression coefficients are
+larger with multiple imputation than with single imputation.
+
+In Example 2 we show how to set up a prediction model using multiple
+imputation. We combine the predictions from multiple imputations and show that
+this results in better regression performance.
+"""
+print(__doc__)
+
+# Authors: Rianne Schouten <r.m.schouten@uu.nl>
+#          Sergey Feldman <sergeyfeldman@gmail.com>
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import special
+
+from sklearn.datasets import load_boston
+from sklearn.linear_model import BayesianRidge
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from sklearn.impute import IterativeImputer
+from sklearn.metrics import mean_squared_error as mse
+
+rng = np.random.RandomState(0)
+
+
+def ampute(X, missing_rate=0.75, strategy="MCAR"):
+    """Insert missing data into X.
+
+    Ampute is the inverse of impute and serves at simulating a dataset with
+    missing data. Two strategies are implemented to remove data: (1) missing
```

Comment:
not sure 'strategy' is the right word here. How about 'missing data mechanism' ?